docs/builtins.md

```diff
@@ -50,20 +50,20 @@ type_name("str") // string
 type_name([1, 2, 3]) // array
 ```
 
+## string
 
-Optionally it can take the second argument, which will be returned if the first argument cannot be converted to string. Note that the second argument does not have to be string.
+Tries to convert an object to string object. See [Runtime Types](https://github.com/d5/tengo/blob/master/docs/runtime-types.md) for more details on type conversion.
 
 ```golang
-v = string(undefined, "foo")  // v == "foo"
-v = string(undefined, false)  // v == false 
+x := string(123) //  v == "123"
```

Comment:
Should be  x == “123”

Reply:
Hey, sorry I never got around to making this change. Thanks for taking care of it. I know it has no effect on the code, but still appreciate the maintenance and effort!